src/sc2utils/sc2_manage_process.cc

```diff
@@ -238,7 +238,7 @@ bool TerminateProcess(uint64_t process_id) {
     if (index < 0)
         return false;
 
-    ::TerminateProcess(windows_processes[index].pi_.hProcess, -1);
+    ::TerminateProcess(windows_processes[index].pi_.hProcess, static_cast<UINT>(-1));
```

Comment:
The actual argument type is UINT, per the [Win32 docs for TerminateProcess](https://msdn.microsoft.com/en-us/library/windows/desktop/ms686714(v=vs.85).aspx). Additionally, it appears this code is protected by the `#ifdef _WIN32` preprocessor macro, so I don't expect it should be used in clang/gcc (at least under Linux/OSX).